src/style.css

```diff
@@ -0,0 +1,106 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  color: #3d4e61;
+  font-weight: 300;
+}
+
+.search-image-container {
+  position: relative;
+}
+
+.search-image-container::before {
+  content: "";
+  position: absolute;
+  background: url(images/Search.svg) no-repeat;
+  top: 25px;
+  bottom: 0;
+  left: 26px;
+  margin: auto 0;
+  width: 25px;
+}
+
+.search-bar-big {
+  margin-top: 20px;
+}
+
+.search-field-big {
+  height: 70px;
+  width: 100%;
+  border-radius: 4px;
```

Comment:
Try to organize your styles
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)